### PR TITLE
Added support for Dutch, slightly increased general localisation

### PR DESCRIPTION
**Pull request by VidalSpierings**
- Created localisation for Dutch by adding localised strings.xml file in newly created values-nl map under res and localising arrays.xml file to Dutch

- Slightly improved localisation bij partially localising arrays.xml file to all languages with Latin alphabet that were already localised

- - -
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/6151)

### CODE REVIEW

File: src/android/app/src/main/res/values-it/arrays.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <item>Cinese Semplificato (简体中文)</item>
+        <item>Coreano (한국어)</item>
+        <item>Neerlandese (Nederlands)</item>
+        <item>Portoghese (português)</item>
+        <item>Russo (Русский)</item>
+        <item>Cinese Tradizionale (正體中文)</item>
+    </string-array>
+
+    <string-array name="cameraFlipNames">
+        <item>Nessuno</item>
```
Comment (Fs00) [inline src/android/app/src/main/res/values-it/arrays.xml:33]:
```suggestion
        <item>Nessuna</item>
```

File: src/android/app/src/main/res/values-it/arrays.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <item>Orizzontale</item>
+        <item>Verticale</item>
+        <item>Inversione</item>
```
Comment (Fs00) [inline src/android/app/src/main/res/values-it/arrays.xml:36]:
```suggestion
        <item>Invertita</item>
```

**Review by Fs00 — CHANGES_REQUESTED** [review]
There a few issues in the Italian translation

**Comment by SachinVin** [comment]
Thanks, but translations are currently managed on transifex, https://explore.transifex.com/citra/citra/ .
Cc: @FearlessTobi

**Pull request closed without merging**